contrib/traceroute/ifaddrlist.c

```diff
@@ -141,7 +141,7 @@ ifaddrlist(register struct ifaddrlist **ipaddrp, register char *errbuf)
 			continue;
 
 
-		(void)strncpy(device, ifr.ifr_name, sizeof(ifr.ifr_name));
+		(void)strncpy(device, ifr.ifr_name, sizeof(device) - 1);
```

Comment:
if you are changing this, you'd be better off using strlcpy and nerfing the next line.